docs/adminguide/setup.rst

```diff
@@ -0,0 +1,126 @@
+******************
+Installation/Setup
+******************
+
+Initial setup (development)
+###########################
+
+1. Prepare your local system:
+    .. code-block::
+
+        $ sudo zypper in python3-pip python3-virtualenv python3-devel libopenssl-devel
```

Comment:
You should also install the package `openldap2-devel` as the header file `lber.h` is needed:

```suggestion
        $ sudo zypper in python3-pip python3-virtualenv python3-devel libopenssl-devel openldap2-devel
```

Reply:
Done.